### ISSUE TITLE
Persona 3 Portable stuck on 11/9 after school loading screen

### ISSUE THREAD

**Issue by Herrscher-of-Yuri** [hrydgard/ppsspp#19621]
### Game or games this happens in

ULUS10512 : Persona3 PORTABLE

### What area of the game / PPSSPP

on 11/9 (the day Ryoji joins as a transfer student) after the teacher speaks it goes from Afternoon to Afterschool but gets stuck on the fully black loading screen. Music is still playing though. Ive also tried restarting the game, my laptop, I even went back and beat the entire game up to 11/9 again and it still gets stuck
![ULUS10512_00005](https://github.com/user-attachments/assets/3ba2ed7c-72a0-4616-82bc-583c50254453)

### What should happen

it should go to Afterschool  a short black loading screen then allow me to do afterschool  activity eg: a social link, then the day should end

### Platform

Windows

### Mobile device model or graphics card (GPU)

Intel(R) Celeron(R) N4120 CPU

### PPSSPP version affected

V1.10.3

### Graphics backend (3D API)

Other

### Checklist

- [X] Test in the [latest git build](https://www.ppsspp.org/devbuilds) in case it's already fixed.
- [X] [Search for other reports](https://github.com/hrydgard/ppsspp/search?q=my+issue&type=issues) of the same issue.
- [X] Try resetting settings or older versions and include if the issue is related.
- [X] Try without any cheats and without loading any save states.
- [X] Include logs or screenshots of issue.

**Comment by hrydgard** [comment]
1.10.3 is ancient. Does it happen in the latest version of PPSSPP?

**Comment by Herrscher-of-Yuri** [comment]
it does only other version i could find was V1.18.1 and it does this issue on that as well

**Comment by anr2me** [comment]
Have you tried changing the I/O Timing method on System setting?

**Comment by Herrscher-of-Yuri** [comment]
I tried all of them none fixed it

**Comment by Ir-boop** [comment]
@Herrscher-of-Yuri can you share the save file before the issue happens?

**Comment by Herrscher-of-Yuri** [comment]
@Ir-boop sorry but how do I do that? I tried putting the file in here and it said I couldnt

**Comment by Ir-boop** [comment]
Try compress the save folder to a zip file,... The try again... Or try host on something like google drive and copy the link here.

**Comment by Herrscher-of-Yuri** [comment]
[ULUS10512DATA00.zip](https://github.com/user-attachments/files/17723637/ULUS10512DATA00.zip)
Does this work? Zipped and copied though not sure if I did it right never had to do this sorry  @Ir-boop

**Comment by Ir-boop** [comment]
Nah, it's fine. I'm testing it and indeed blackscreen happens to me too when the game load 'after school' section on that date. I wonder what's the issue is.

**Comment by sum2012** [comment]
Can someone clearly how to reproduce ?

**Comment by sum2012** [comment]
I have studied  in library , nothing wrong in evening.
![3](https://github.com/user-attachments/assets/b75dadb2-c33c-451c-ad7b-606f694c3e7f)

**Comment by Herrscher-of-Yuri** [comment]
@sum2012 its on 11/9 not 11/7

**Comment by sum2012** [comment]
No "Info log before hang"
hard to know the reason
![4](https://github.com/user-attachments/assets/d19a2772-f216-4089-bc12-2d102a4e1124)
